packages/lablqml/lablqml.0.6.2/opam

```diff
@@ -1,11 +1,16 @@
 opam-version: "2.0"
 maintainer:   "kakadu.hafanana@gmail.com"
 authors:      "kakadu.hafanana@gmail.com"
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
 homepage:     "http://kakadu.github.io/lablqt/"
 bug-reports:  "https://github.com/kakadu/lablqml/issues"
 dev-repo: "git+https://github.com/Kakadu/lablqml.git"
 tags: [ "gui" "ui" "qt" ]
 
+# Marking as unavailable because opam 2.1 sandbox doesn't allow creating new files this way
+# As we have version of lablqml newer then 0.6.2, this issue is not going to be fixed.
+available: false
```

Comment:
This is going to work with older opam versions right? If so, I would keep it available. @kit-ty-kate is there a way to set availability depending on opam?

Reply:
Yes, AFAIU it should work with older opam. The old release definitely passed CI some time ago. 
But because of old version is not widely used (never got any feedback about this), I think it is OK to mark it as unavailable.

Reply:
To be frank, I would keep it available but add a postinstall-failure message saying that you need to disable the sandbox for this version to work. Same for qocamlbrowser

Reply:
I applied changes that you requested.